app/cdap/components/WrangleHome/Components/OngoingDataExplorationCard/index.tsx

```diff
@@ -70,14 +70,26 @@ export default function OngoingDataExplorationCard({ item }) {
             );
           case 'text':
             return eachItem.label.includes('Recipe steps') ? (
-              <Grid item xs={3} className={classes.elementStyle} key={index}>
+              <Grid
```

Comment:
What is the difference between these two ternary return values? Also why is there a ternary here if there is a small difference just make the small difference be the ternary.

Reply:
this file is not specific to select column panel feature and this file is updated  in it's corresponding feature PR
which is not yet merged

Reply:
If that is the case next time please note the areas that *shouldn't* be looked at